lib/l10n/intl_en.arb

```
@@ -416,38 +407,36 @@
   "pleaseEnterLightningInvoiceFor": "Please enter a Lightning Invoice for: ",
   "sats": " sats",
   "lightningInvoice": "Lightning Invoice",
   "enterInvoiceHere": "Enter invoice here",
   
-  "@_comment_trade_detail_screen": "Trade Detail Screen Strings",
-  "cancelPending": "CANCEL PENDING",
-  "acceptCancel": "ACCEPT CANCEL",
-  "completePurchase": "COMPLETE PURCHASE",
-  "rate": "RATE",
-  "contact": "CONTACT",
+  "cancelPendingButton": "CANCEL PENDING",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)